patches/gcc/0002-Managarm-target.patch

```diff
@@ -87,6 +87,10 @@ index ddd3b8f4d..9701267b9 100644
 +    tmake_file="${tmake_file} t-slibgcc"
 +    thread_file='posix'
 +    ;;
++  *-managarm-pc*)
++    tmake_file="${tmake_file} t-slibgcc"
++    thread_file='posix'
++    ;; 
```

Comment:
Why do we need `managarm-pc`?

Reply:
(Not saying that it's wrong, I'm just surprised that another suffix is needed.)

Reply:
Gcc defaults to this suffix with the current configuration options. We can probably get it to accept managarm-system there as well but something-pc is pretty universal as a suffix

Reply:
OK.